docs/geedocs/docsrc/answer/fusionAdministration/confTaskRulesForFusionPerf.rst

```diff
@@ -410,10 +410,24 @@ Configure Fusion performance
       - ``ALL_GRAPH_OPS`` or ``2``: use all new operations regardless of
         performance impact
 
+      .. _Use_Minification:
+      .. rubric:: Use Minification
+
+      Included in ``/gevol/assets/.config/misc.xml`` is a ``UseMinification``
+      option that controls how data for imagery and terrain zoom levels are
+      generated.  ``UseMinification`` can be set to the following values:
+
+      - ``1``: Imagery and terrain for each zoom level are generated by
+        minifying the level below it.  This is the default.
```

Comment:
Might be nice to add "where possible" to the end of the first sentence. Even if "UseMinification" is on, it won't minify for levels higher than the lowest zoom level of any image in the project.